components/carousel.css

```diff
@@ -0,0 +1,81 @@
+.carousel {
+    height: 600px;
+    width: 80%;
+    margin: 0 auto;
+    position: relative;
+}
+
+.carousel__image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.carousel__track-container {
+    position: relative;
```

Comment:
you don't need to define multiple times of same property you can write it like that:-
```
.carousel, .carsousel__track-container{
position:relative;
}
```
and similar for other properties that is used multiple times.

Reply:
updated it